pkg/artifactory/resource/repository/remote/resource_artifactory_remote_repository_test.go

```diff
@@ -22,6 +22,49 @@ import (
 	"github.com/jfrog/terraform-provider-shared/validator"
 )
 
+func TestAccRemoteUpgradeFromSDKv2(t *testing.T) {
```

Comment:
Why is this test necessary?

Reply:
I wanted to see how it will behave if updated from the version without that attribute to the version with the attribute with the default value.